input_output/file_output.py

```diff
@@ -0,0 +1,44 @@
+from supporting_components.graph_io import load_graph, write_dot
+import subprocess
+
+
+"""
+Before you can use this module you must install graphviz on the terminal to be able to do this:
+    https://stackoverflow.com/questions/1494492/graphviz-how-to-go-from-dot-to-a-graph
+Install instructions:
+    https://www.graphviz.org/download/
+"""
+
+def load_graph_list(filename):
```

Comment:
Je voegt hier nu de hele file toe, terwijl het eigenlijk een aanpassing is aan de file die nu al in de master zit. Het zal met mergen waarschijnlijk wel goed gaat, maar lijkt minder mooi in de commits.